sources/packages/api/src/route-controllers/institution-locations/models/institution-location.dto.ts

```diff
@@ -32,6 +32,31 @@ export class InstitutionLocationFormAPIInDTO extends AddressDetailsAPIInDTO {
   primaryContactPhone: string;
 }
 
+export class InstitutionLocationPrimaryContactAPIInDTO {
+  @Allow()
+  primaryContactFirstName: string;
+  @Allow()
+  primaryContactLastName: string;
+  @Allow()
+  primaryContactEmail: string;
+  @Allow()
+  primaryContactPhone: string;
+}
+export class AESTInstitutionLocationAPIInDTO extends AddressDetailsAPIInDTO {
```

Comment:
Are we executing the `dryRun` for this submission? If not we need to add the proper validators.